docs/en_US/TrainingService/PaiMode.md

```diff
@@ -82,6 +82,22 @@ Compared with [LocalMode](LocalMode.md) and [RemoteMachineMode](RemoteMachineMod
         portNumber: 1
     ```
 
+NNI support two kind of authorization method in PAI, including password and AAD token (could be viewed from PAI webportal). The authorization is configured in `paiConfig` field.  
```

Comment:
"could be viewed from PAI webportal" --> could you add a link here.

Reply:
Updated